iter-1/src/main/java/com/github/lette1394/App.java

```diff
@@ -1,11 +1,29 @@
 package com.github.lette1394;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 import java.util.Scanner;
 
 public class App {
+
   public static void main(String[] args) {
+
     final Scanner sc = new Scanner(System.in);
-    sc.nextLine();
-    System.out.println(5);
+    final String getOperator = sc.nextLine();
+
+    final List<String> expression = new ArrayList<>(Arrays.asList(getOperator.split("\\+")));
+    final long operand1 = Long.parseLong(expression.get(0));
+    final long operand2 = Long.parseLong(expression.get(1));
+    final long operand3 = Long.parseLong(expression.get(2));
+    if (expression.size() == 2) {
```

Comment:
빈 줄 하나 넣어주면 좋을 거 같아요

```suggestion
    final long operand3 = Long.parseLong(expression.get(2));
    
    if (expression.size() == 2) {
```